easy_phi/app.py

```diff
@@ -124,11 +126,11 @@ def get(self):
 
 # URL schemas to RequestHandler classes mapping
 APPLICATION = tornado.web.Application([
-    (r"/", VersionHandler),  # Default page
-    (r"/version", VersionHandler),
-    (r"/modules", ModulesListHandler),
-    (r"/pick_module", SelectModuleHandler),
-    (r"/send_command", SCPICommandHandler),
+    (r"/", PlatformInfoHandler),  # Default page
```

Comment:
default page is web interface, not an API call. Here it is handled by the same view as api call in next line